src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -1253,10 +1255,18 @@ static ssize_t queue_write_iter(struct kiocb *kiocb, struct iov_iter *io)
 	}
 
 	if (!is_sync_kiocb(kiocb)) {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,4,0)
```

Comment:
fix format in rest of files also for KERNEL_VERSION

Reply:
@rbramand-xilinx I don't see the any format difference for KERNEL_VERSION with the existing code in 2021.1 branch. Can you summarize more if we are missing anything?

Reply:
use KERNEL_VERSION(6, 4, 0)  -> I mean use space after comma's. Its trivial but good to have